lib/super_digit.rb

```diff
@@ -1,15 +1,38 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n)
 def super_digit(n)
```

Comment:
Just a note that both `n.digits` and `n_digits.sum` are O(n) operations and since you have this in a recursive call...